packages/neuron-ui/package.json

```diff
@@ -53,6 +53,7 @@
     "@nervosnetwork/ckb-sdk-core": "0.109.0",
     "@nervosnetwork/ckb-sdk-utils": "0.109.0",
     "canvg": "2.0.0",
+    "ckb-walletconnect-wallet-sdk": "0.0.1-alpha.9",
```

Comment:
Maybe it would be more convenient to move the SDK to this repository and use the yarn workspace feature, or import the SDK via git submodule. This would simplify development as it would not require publishing every change made to the SDK.